grakn/rpc/transaction.py

```diff
@@ -45,14 +45,14 @@ class TransactionType(enum.Enum):
 
 class Transaction:
 
-    def __init__(self, channel: grpc.Channel, session_id: str, transaction_type: TransactionType, options=GraknOptions()):
+    def __init__(self, address: str, session_id: str, transaction_type: TransactionType, options=GraknOptions()):
         self._transaction_type = transaction_type
         self._concept_manager = ConceptManager(self)
         self._query_manager = QueryManager(self)
         self._logic_manager = LogicManager(self)
         self._response_queues = {}
 
-        self._grpc_stub = GraknStub(channel)
+        self._grpc_stub = GraknStub(grpc.insecure_channel(address))
```

Comment:
This is the main fix here: it seems that using a single gRPC Channel was causing a potential issue where responses from the server would be dropped leading to stalling.

Now, we create a separate Channel for each transaction (and session)